pynamodb/connection/base.py

```diff
@@ -1127,8 +1103,6 @@ def rate_limited_scan(self,
                           attributes_to_get=None,
                           page_size=None,
                           limit=None,
-                          conditional_operator=None,
```

Comment:
I think we can just remove `rate_limited_scan` in favor of the paginators.

Reply:
TIL!